src/main/java/com/spikes2212/command/drivetrains/commands/DriveTank.java

```diff
@@ -6,8 +6,10 @@
 import java.util.function.Supplier;
 
 /**
+ * @author
  * This command moves a {@link TankDrivetrain} using speeds supplied to the left and the right sides independently.
  */
+
 public class DriveTank extends CommandBase {
 
     protected final TankDrivetrain tankDrivetrain;
```

Comment:
I cant comment below so:
on line 18: add visibility to `isFinished`

Reply:
a mistake